src/main/java/net/dancier/domain/dance/School.java

```diff
@@ -0,0 +1,19 @@
+package net.dancier.domain.dance;
+
+import lombok.Builder;
+import lombok.Data;
+import net.dancier.domain.Recommendable;
+import net.dancier.domain.User;
+
+import java.net.URL;
+
+@Data
+@Builder
+public class School implements Recommendable {
+    private User user;
+
+    private String name;
```

Comment:
You should document (best with NotNull, NotrEmpty or NonNull annoations) what is

Reply:
Did not get what you want, maybe we have a chat tomorrow or whatever.

Reply:
Did it